WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeCreateCampaignIntroView.swift

```diff
@@ -0,0 +1,233 @@
+
+/// Hosting controller for `BlazeCreateCampaignIntroView`.
+///
+final class BlazeCreateCampaignIntroController: UIHostingController<BlazeCreateCampaignIntroView> {
+    init(onCreateCampaign: @escaping () -> Void,
+         onDismiss: @escaping () -> Void) {
+        super.init(rootView: BlazeCreateCampaignIntroView(onCreateCampaign: onCreateCampaign,
+                                                          onDismiss: onDismiss))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// View to display the introduction to the Blaze campaign creation
+///
+struct BlazeCreateCampaignIntroView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    private let onCreateCampaign: () -> Void
+    private let onDismiss: () -> Void
+
+    private let features: [Feature] = [.init(title: Localization.AudienceFeature.title, subtile: Localization.AudienceFeature.subtitle),
+                                       .init(title: Localization.GlobalReach.title, subtile: Localization.GlobalReach.subtitle),
+                                       .init(title: Localization.QuickStart.title, subtile: Localization.QuickStart.subtitle)]
+
+    init(onCreateCampaign: @escaping () -> Void,
+         onDismiss: @escaping () -> Void) {
+        self.onCreateCampaign = onCreateCampaign
+        self.onDismiss = onDismiss
+    }
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Button(action: {
+                onDismiss()
+            }, label: {
+                Image(uiImage: .closeButton)
+                    .secondaryBodyStyle()
+            })
+            .padding(Layout.closeButtonPadding)
+
+            ScrollView {
+                VStack {
+                    HStack(spacing: 8) {
+                        Spacer()
+
+                        Image(uiImage: .blaze)
+                            .resizable()
+                            .renderingMode(.template)
+                            .foregroundColor(Color(.brand))
+                            .frame(width: Layout.logoSize * scale, height: Layout.logoSize * scale)
+
+                        Text(Localization.headline)
+                            .foregroundColor(Color(.brand))
+                            .fontWeight(.semibold)
+                            .headlineStyle()
+
+                        Spacer()
+                    }
+
+                    VStack(spacing: Layout.titleElementsVerticalSpacing) {
+                        Text(Localization.title)
+                            .fontWeight(.bold)
+                            .largeTitleStyle()
+                            .multilineTextAlignment(.center)
+
+                        Text(Localization.subtitle)
+                            .foregroundColor(Color(.secondaryLabel))
+                            .bodyStyle()
```

Comment:
Nit: we can simply use `.secondaryBodyStyle()` for this.

Reply:
Done in 061ce49dd51056c6bbe5dd808cf8cd6b75799cc2